freqtrade/arguments.py

```diff
@@ -306,6 +306,14 @@ def hyperopt_options(parser: argparse.ArgumentParser) -> None:
             dest='print_all',
             default=False
         )
+        parser.add_argument(
+            '--random-state',
+            help='Set random state to some positive integer for reproducible hyperopt results.',
+            dest='hyperopt_random_state',
+            default=None,
+            type=Arguments.check_int_positive,
```

Comment:
THIS is a possibility? :open_mouth: great!

Reply:
https://stackoverflow.com/questions/14117415/in-python-using-argparse-allow-only-positive-integers